protocol/src/main/java/org/apache/drill/exec/proto/beans/RpcType.java

```diff
@@ -25,28 +25,8 @@
     HANDSHAKE(0),
     ACK(1),
     GOODBYE(2),
-    RUN_QUERY(3),
-    CANCEL_QUERY(4),
-    REQUEST_RESULTS(5),
-    RESUME_PAUSED_QUERY(11),
-    GET_QUERY_PLAN_FRAGMENTS(12),
-    GET_CATALOGS(14),
-    GET_SCHEMAS(15),
-    GET_TABLES(16),
-    GET_COLUMNS(17),
-    CREATE_PREPARED_STATEMENT(22),
-    GET_SERVER_META(8),
-    QUERY_DATA(6),
-    QUERY_HANDLE(7),
-    QUERY_PLAN_FRAGMENTS(13),
-    CATALOGS(18),
-    SCHEMAS(19),
-    TABLES(20),
-    COLUMNS(21),
-    PREPARED_STATEMENT(23),
-    SERVER_META(9),
-    QUERY_RESULT(10),
-    SASL_MESSAGE(24);
+    REQ_RECORD_BATCH(3),
+    SASL_MESSAGE(4);
```

Comment:
Is this your change? Why has the list of RPC types changed? Is this a merge issue?

Reply:
This is not my change. This file is generated by protobuf. When I looked into the issue I found out that user.proto and BitData.proto, Bitcontrol.proto have same enum (RpcType) . That is causing the issue.

Reply:
The change seems to be that messages are dropped. That can't be good. The only diff that should show up here is the addition of your new state codes.

The other explanation is that master is wrong, which would be a bad state of affairs.